odoo/addons/base/models/ir_model.py

```diff
@@ -1162,6 +1176,42 @@ def _add_manual_fields(self, model):
                 except Exception:
                     _logger.exception("Failed to load field %s.%s: skipped", model._name, field_data['name'])
 
+    @api.model
```

Comment:
These are mostly used by fields_get; I believe we should change the implementation of fields_get to retrieve `string` and `help` at once (and may be `selection`?), instead of querying ir.model.fields for each attribute.

We might also avoid the `search` if the lang is en_US, and retrieve the data for the field in memory instead.